pkg/minishift/config/validations.go

```diff
@@ -166,7 +166,7 @@ func IsValidISOUrl(_ string, isoURL string) error {
 		}
 	}
 
-	return nil
+	return fmt.Errorf("'%s' file is not present", isoURL)
```

Comment:
@anjannath return error at the end of the function doesn't something I expect this should be handled soon and returned.

why not having ?

```golang
        if runtime.GOOS == "windows" {
 		match, _ := regexp.MatchString("^file://[a-zA-Z]:/.+", isoURL)
 		if !match {
 			return fmt.Errorf("'%s' url is not valid", isoURL)
 		}
 		if !filehelper.Exists(strings.Replace(strings.TrimPrefix(isoURL, "file://"), "/", "\\", -1)) {
 			return fmt.Errorf("'%s' file is not present", isoURL)
 		}
 	} else {
 		match, _ := regexp.MatchString("^file:///.+", isoURL)
 		if !match {
 			return fmt.Errorf("'%s' url is not valid", isoURL)
 		}
 		if !filehelper.Exists(strings.TrimPrefix(isoURL, "file://")) {
 			return fmt.Errorf("'%s' file is not present", isoURL)
		}
	}
return nil
```